src/Components/Samples/BlazorUnitedApp/Pages/Counter.razor

```diff
@@ -1,5 +1,5 @@
 ﻿@page "/counter"
-
+@attribute [RenderModeServer]
```

Comment:
Assuming this doesn't actually work yet (we don't have the JS parts) would be good to revert this to keep us focused. I expect you just added it for testing.

Reply:
Should work if you add the server script, but I've removed in the source generator PR.